### PR TITLE
Allow to start manual transaction only if auto transactions are disab…

### PR DESCRIPTION
**Pull request by taburet**
…led in graph database #5865

### CODE REVIEW

File: graphdb/src/main/java/com/tinkerpop/blueprints/impls/orient/OrientBaseGraph.java [hunk 1]
```diff
@@ -1738,7 +1738,7 @@ public long countEdges(final String iClassName) {
             "Requested command '%s' must be executed outside active transaction: the transaction will be committed and reopen right after it. To avoid this behavior execute it outside a transaction",
             msg.toString());
       }
-      raw.commit();
+      raw.commit(true);
```
Comment (taburet) [inline graphdb/src/main/java/com/tinkerpop/blueprints/impls/orient/OrientBaseGraph.java]: Not sure about this, before nested tx support this was relatively safe operation, now we need to commit all nested transactions too, but we can't restart them.

Comment (laa) [inline graphdb/src/main/java/com/tinkerpop/blueprints/impls/orient/OrientBaseGraph.java]: Why we can not ? We can , merely call begin the same amount of times as amount of nested txs was, was not it ?

Comment (taburet) [inline graphdb/src/main/java/com/tinkerpop/blueprints/impls/orient/OrientBaseGraph.java]: Good point, will fix it.

**Comment by laa** [comment]
Will wait for fixed one.

**Pull request closed without merging**